lab-davide/test/greet-test.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+const greeting = require(__dirname + '/../lib/greet');
```

Comment:
in the future would name this variable the same as the file name,, like you did in hello.js 
:)